utils/storage_utils.py

```diff
@@ -75,6 +75,11 @@ def write_output_csv(start_time, end_time, cve_model_type, ecosystem, df, s3_upl
     return df
 
 
+def _restrict_data_length(data: str) -> str:
+    """Ristrict long length data content."""
+    return data[0:oc.MAX_STRING_LEN_FOR_CSV_EXPORT] if data else data
```

Comment:
Else case here will return `None` only, better to return the same explicitly.

Reply:
Earlier code was similar only, but we were getting `"'NoneType' object is not subscriptable"`, due to None data so  added this condition to handle the same.